### PR TITLE
Allow multiple node allocation

### LINKED ISSUE #15234: Modify TestHarness to allow selection of multiple nodes
## Reason
One of the Grizzly nightly assessment cases requires more than one node to run, but currently the TestHarness does not have the ability to select more than one node. 

## Design
Allow a new option to inform the TestHarness how many cores are available per node. It could do simple math and split up a request with more cores than can fit on a single node. 

## Impact
Will enable testing of larger problems.

### PR DESCRIPTION
**Pull request by milljm**
Some projects are asking for more cores than what is available on a single
node during testing with PBS options.

This PR allows a user to supply the core count of each individual node for
their cluster. Using math.ceil, we round up a whole number to accommodate an
additional node than otherwise allowed with an over-subscribing min_parallel
setting.

Usage:

```bash
./run_tests --pbs session --pbs-node-cpus 32
```

Closes #15234

### CODE REVIEW

File: python/TestHarness/schedulers/RunPBS.py [hunk 1]
```diff
@@ -80,6 +81,13 @@ def _augmentTemplate(self, job):
         # NCPUS
         template['mpi_procs'] = job.getMetaData().get('QUEUEING_NCPUS', 1)
 
+        # Compute node requirement
+        if self.options.pbs_node_cpus:
+            _ceiling = template['mpi_procs']/self.options.pbs_node_cpus
+        else:
+            _ceiling = 1
+        template['nodes'] = math.ceil(_ceiling)
+
```
Comment (loganharbour) [inline python/TestHarness/schedulers/RunPBS.py:90]:
```suggestion
        # Compute node requirement
        if self.options.pbs_node_cpus:
            ceiling = template['mpi_procs']/self.options.pbs_node_cpus
        else:
            ceiling = 1
        template['nodes'] = math.ceil(ceiling)

```

**Review by loganharbour — CHANGES_REQUESTED** [review]

File: python/TestHarness/schedulers/RunPBS.py (hunk 1 above)
Comment (loganharbour) [reply python/TestHarness/schedulers/RunPBS.py:90]: Removes the `_` prefix for `_ceiling`

**Review by loganharbour — APPROVED** [review]

**Comment by bwspenc** [comment]
Looks like this is passing and has been approved. Any reason we shouldn't merge it now?

**Comment by milljm** [comment]
I thought it did get merged. Guess not!

**Pull request merged**